Feat/calc 03 historique des operations

juloStud opened the pull request.

juloStud committed:
affiche l'historique des calculs

affiche le panneau contenant l'historique des calculs effectués

juloStud committed:
Merge branch 'develop' into feat/CALC-03-historique-des-operations

# Conflicts:
#	index.js
#	style.css

juloStud committed:
corrige l'historique des calculs

corrige l'historique des calculs effectués

Alandla commented:
Git:
- Résoudre le conflits
- Ajouter le .idea au .gitignore

Design:
- Justifier le texte de l'historique vers la droite
- Remettre le dernier résultat au bon endroit
- Mettre le dernier calcul de l'historique en plus grand, et les anciens en plus petit et gris clair

juloStud committed:
Merge branch 'develop' into feat/CALC-03-historique-des-operations

juloStud committed:
modifie l'affichage de l'historique des calculs

modifie l'affichage dans l'historique des calculs en mettant en évidence le dernier calcul et en grisant les autres

Alandla commented:
- Ligne 54 enlever le commentaire
- Faire en sorte que le bouton 'C' clear aussi l'historique

juloStud committed:
ajoute la suppression de l'historique sur le bouton C

Supprime l'historique en même temps que la calculatrice se réinitialise
